bdk-swift/build-local-swift.sh

```diff
@@ -13,8 +11,8 @@ rustup target add aarch64-apple-ios-sim  # simulator mac M1
 rustup target add aarch64-apple-darwin   # mac M1
 rustup target add x86_64-apple-darwin    # mac x86_64
 
-pushd bdk-ffi
-mkdir -p Sources/BitcoinDevKit
+cd ../bdk-ffi/
+# mkdir -p temp/Sources/BitcoinDevKit
```

Comment:
I don't think this line did anything because the directory is not referenced anywhere after.